core/src/main/scala/org/apache/spark/executor/Executor.scala

```diff
@@ -613,7 +614,7 @@ private[spark] class Executor(
     private[this] val taskId: Long = taskRunner.taskId
 
     private[this] val killPollingIntervalMs: Long =
-      conf.getTimeAsMs("spark.task.reaper.pollingInterval", "10s")
+      conf.getTimeAsSeconds("spark.task.reaper.pollingInterval", "10s").seconds.toMillis
```

Comment:
Actually I believe this is change things from right to wrong. If you want to use a second value then you shall use sth like `1s`.